MAUI/Calendar/selections.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: post
+title: Selections in MAUI Calendar widget | Syncfusion
+description: Learn here all about Selections feature in Syncfusion MAUI Calendar (SfCalendar) widget and more.
+platform: maui
+control: SfCalendar
+documentation: ug
+---
+
+# Selections in MAUI Calendar (SfCalendar)
+The Selection mode is specified in the calendar property enumeration. Can select the dates by tap the cell in the calendar. `SfCalendar` provides three types mode such as `Single`, `Multiple` and `Range` Selection. The default `SelectionMode` is Single that allows the user to select one date at a time.
+
+>**NOTE** Can able to select the cells in the Year, Decade and Century views in the calendar only when the `AllowViewNavigation` property is set to false.
+
+## Single selection
+The `Single` selection can be performed in the Calendar by setting the `CalendarSelectionMode` property to `Single`. In this Selection, you can able to select a single date from the cell.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+<calendar:SfCalendar  x:Name="Calendar" 
+                        View="Month"
+                        SelectionMode="Single">
+</calendar:SfCalendar>
+
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+
+this.Calendar.SelectionMode = CalendarSelectionMode.Single;
+
+{% endhighlight %}
+{% endtabs %}
+
+![Month view single selection in .NET MAUI Calendar.](images/selection/maui-month-view-single-selection.png)
+{% endcolumn %}
+
+>**NOTE**
+* In the Year, Decade and Century views can able to select the cells, only when the `AllowViewNavigation` property is set to false.
```

Comment:
* In the Year, Decade, and Century views, you can select the cells, only when the `AllowViewNavigation` property is set to false.